### PR TITLE
docs: remove outdated screenshots

### PR DESCRIPTION
**Pull request by nsprenkle**
**TL;DR -** Remove out-of-date screenshots from README

JIRA: [AU-60](https://openedx.atlassian.net/browse/AU-60)

**What changed?**

- We've made a bunch of changes to Gradebook. To keep the docs from getting stale, just remove screenshots entirely. Gradebook functionality is documented textually, and with screenshots in https://github.com/edx/edx-documentation where important.

**Developer Checklist**
- [x] Test suites passing
- [x] Documentation and test plan updated, if applicable
- [x] Received code-owner approving review
- [ ] Bumped version number [package.json](../package.json)

**Testing Instructions**

README.md no longer has images... 

**Reviewer Checklist**

Collectively, these should be completed by reviewers of this PR:

- [x] I've done a visual code review
- [x] I've tested the new functionality

FYI: @edx/masters-devs-gta

### CODE REVIEW

**Review by jansenk — APPROVED** [review]

**Review by mattcarter — APPROVED** [review]
removed and approved 👍

**Review by leangseu-edx — APPROVED** [review]
